classes/core-image-pelux.bbclass

```diff
@@ -6,7 +6,7 @@
 
 inherit core-image
 
-IMAGE_FEATURES += "package-management debug-tweaks"
+IMAGE_FEATURES_append = " package-management debug-tweaks"
```

Comment:
Is this due to something similar to https://github.com/Pelagicore/meta-pelux/pull/426 ?

Reply:
Yes.

Although I do not remember if it was a problem, or if it was for future-proofing / making it look like other places where we have changed to use _append instead of += ...

Most likely the latter.

Reply:
OK, maybe just mention "+= -> _append for IMAGE_FEATURES to be consistent" in commit message.